scheduler-driver/src/main/java/gov/nasa/jpl/aerie/scheduler/solver/PrioritySolver.java

```diff
@@ -947,52 +946,16 @@ private Optional<SchedulingActivityDirective> instantiateActivity(
-    final var taskNetwork = new TaskNetworkAdapter(new TaskNetwork());
-    taskNetwork.addAct(name);
-    if (interval != null) {
-      taskNetwork.addEnveloppe(name, "interval", interval.start, interval.end);
-    }
-    taskNetwork.addEnveloppe(name, "planningHorizon", planningHorizon.getStartAerie(), planningHorizon.getEndAerie());
-    if (activityExpression.startRange() != null) {
-      taskNetwork.addStartInterval(name, activityExpression.startRange().start, activityExpression.startRange().end);
-    }
-    if (activityExpression.endRange() != null) {
-      taskNetwork.addEndInterval(name, activityExpression.endRange().start, activityExpression.endRange().end);
-    }
-    Optional<Duration> durRequirementLower = Optional.empty();
-    Optional<Duration> durRequirementUpper = Optional.empty();;
-    if (activityExpression.durationRange() != null) {
-      try {
-        durRequirementLower = activityExpression.durationRange().getLeft()
-                                                                         .evaluate(null, planningHorizon.getHor(), evaluationEnvironment)
-                                                                         .valueAt(Duration.ZERO)
-                                                                         .flatMap($ -> $.asInt().map(i -> Duration.of(i, Duration.MICROSECOND)));
-        durRequirementUpper = activityExpression.durationRange().getRight()
-                                                                         .evaluate(null, planningHorizon.getHor(), evaluationEnvironment)
-                                                                         .valueAt(Duration.ZERO)
-                                                                         .flatMap($ -> $.asInt().map(i -> Duration.of(i, Duration.MICROSECOND)));
-
-      } catch (NullPointerException e) {
-        throw new UnsupportedOperationException("Activity creation duration arguments cannot depend on simulation results.", e);
-      }
-      if(durRequirementLower.isPresent() && durRequirementUpper.isPresent()) {
-        taskNetwork.addDurationInterval(name, durRequirementLower.get(), durRequirementUpper.get());
-      }
-    }
-    final var success = taskNetwork.solveConstraints();
-    if (!success) {
-      logger.debug("Inconsistent static temporal constraints, cannot place activity in interval");
-      logger.debug("Start range " + activityExpression.startRange());
-      logger.debug("End range " + activityExpression.endRange());
-      logger.debug(
-          "Duration range [" +
-          (durRequirementLower.isPresent() ? durRequirementLower.get() : "-inf") +
-          ", " +
-          (durRequirementUpper.isPresent() ? durRequirementUpper.get() : "+inf"));
-      logger.debug("Interval range " + interval);
-      return Optional.empty();
-    }
-    final var solved = taskNetwork.getAllData(name);
+    final var enveloppes = new ArrayList<Interval>();
```

Comment:
Minor: typo:
```suggestion
    final var envelopes = new ArrayList<Interval>();
```

Reply:
you can find this typo in many of my PRs :D